eng/sendtohelix.proj

```diff
@@ -66,17 +66,24 @@
     <HelixCommand>$(HelixCommand) /p:LocalPackagesPath="%HELIX_CORRELATION_PAYLOAD%\packages\</HelixCommand>
   </PropertyGroup>
 
+  <!-- Define which workloads require global tools. -->
+  <PropertyGroup Condition="'$(UseGlobalTools)' == ''">
+    <UseGlobalTools Condition="'$(CoveraHelixCommandge)' == 'true'">true</UseGlobalTools>
```

Comment:
What is CoveraHelixCommandge? Seems like a typo.

Reply:
oh, thx